src/gluonnlp/data/utils.py

```diff
@@ -407,3 +406,35 @@ def __call__(self, s):
             List of strings. Obtained by calling s.split(separator).
         """
         return s.split(self._separator)
+
+
+def convert_to_unicode(text):
```

Comment:
`isinstance` calls on each string seems expensive. could we narrow down the interface so that it doesn't have to handle several different types?

Reply:
Made it a private function to avoid abuse. For vocab generation the speed is fine.